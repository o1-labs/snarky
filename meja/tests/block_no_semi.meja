let x = fun (f) => {
  f x
};
