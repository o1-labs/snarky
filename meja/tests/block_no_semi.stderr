File "tests/block_no_semi.meja", line 2, characters 4-5:
Error: Missing semicolon.
